io.github.wazzaps.Fingerpaint.yml

```
@@ -0,0 +1,45 @@
+app-id: io.github.wazzaps.Fingerpaint
+runtime: org.gnome.Platform
+runtime-version: '44'
+sdk: org.gnome.Sdk
+command: "/app/bin/fingerpaint"
+finish-args:
+  # GUI Permissions
+  - "--share=ipc"
+  - "--socket=wayland"
+  - "--socket=fallback-x11"
+  - "--device=dri"
+
+  # Access to udev for finding raw touchpad device
+  - "--device=all"
+  - "--filesystem=/sys:ro"
+  - "--filesystem=/run/udev:ro"
+
+  # Access to dconf/gsettings for disabling touchpad under GNOME/Wayland
+  - "--filesystem=xdg-run/dconf:ro"
+  - "--filesystem=~/.config/dconf:ro"
+  - "--talk-name=ca.desrt.dconf"
+  - "--env=DCONF_USER_CONFIG_DIR=~/.config/dconf"
+modules:
+  - generated-poetry-sources.json
+  - name: fingerpaint
+    buildsystem: simple
+    build-commands:
+      - python3 setup.py bdist_wheel && pip3 install --no-deps --no-index --prefix=/app dist/*.whl
+      - install -D flatpak/$FLATPAK_ID.metainfo.xml $FLATPAK_DEST/share/appdata/$FLATPAK_ID.appdata.xml
+      - install -D flatpak/$FLATPAK_ID.desktop $FLATPAK_DEST/share/applications/$FLATPAK_ID.desktop
+      - install -D snap/gui/fingerpaint.svg $FLATPAK_DEST/share/icons/$FLATPAK_ID.svg
```

this is not the correct path for install incons.
```suggestion
      - install -D snap/gui/fingerpaint.svg $FLATPAK_DEST/share/icons/hicolor/scalable/apps/$FLATPAK_ID.svg
```